en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-apis-as-immutable-gateway.md

```diff
@@ -0,0 +1,63 @@
+# Deploy APIs as immutable gateway
```

Comment:
```suggestion
# Deploy APIs as Immutable Gateway
```
Can we reword this title so that users can understand it easily? I'm wondering if "immutable" might be too complex information for a title.

Reply:
Had a discussion with team and decided to move forward with the term Immutable.